src/NServiceBus.Core/Transports/TransportOperations.cs

```diff
@@ -51,11 +51,11 @@ public TransportOperations(params TransportOperation[] transportOperations)
         /// <summary>
         /// A list of multicast transport operations.
         /// </summary>
-        public IEnumerable<MulticastTransportOperation> MulticastTransportOperations { get; }
+        public List<MulticastTransportOperation> MulticastTransportOperations { get; }
```

Comment:
I tried it. It doesn't work because of the TryGet extensions. The ripple
effects were too big

On Friday, July 1, 2016, Tim Bussmann <notifications@github.com> wrote:

> In src/NServiceBus.Core/Transports/TransportOperations.cs
> <https://github.com/Particular/NServiceBus/pull/3893#discussion_r69270212>
> :
>
> > @@ -51,11 +51,11 @@ public TransportOperations(params TransportOperation[] transportOperations)
> >          /// <summary>
> >          /// A list of multicast transport operations.
> >          /// </summary>
> > -        public IEnumerable<MulticastTransportOperation> MulticastTransportOperations { get; }
> > +        public List<MulticastTransportOperation> MulticastTransportOperations { get; }
>
> I think returning arrays would be nicer from an API perspective for
> scenarios where we know the list is "readonly" but I'm ok too with lists
> since this is not on the basic API and it would probably mean additional
> allocations.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3893/files/79d7ac4fd8e885e7aebaec716e6efd3e289d711c#r69270212>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAKosuW2fU5fF67Ooo1X1wfa3sNlxarvks5qRNf-gaJpZM4JCD3c>
> .
>